packages/account/src/utils/Constants.ts

```diff
@@ -1,14 +1,20 @@
 import { ChainId } from '@biconomy/core-types'
+import { EntrypointAddresses, BiconomyFactories, BiconomyImplementation } from './Types'
 
-export const ENTRYPOINT_ADDRESSES = {
-    'V0_0_5': '0x27a4Db290B89AE3373ce4313cBEaE72112Ae7Da9',
-    'V0_0_6': '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789',
-    default: '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789'
+export const ENTRYPOINT_ADDRESSES: EntrypointAddresses = {
+    '0x27a4Db290B89AE3373ce4313cBEaE72112Ae7Da9': 'V0_0_5',
+    '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789': 'V0_0_6',
+    default: '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789' // will always be latest { ENTRYPOINT_ADDRESSES }
```

Comment:
This is not consistent.. for some entries it return the version and for default it returns the address. We should separate these in different data structure.

Reply:
resolved